### DISCUSSION TITLE
Config to disable rendering items in the Tinker's Construct Smeltery

### DISCUSSION THREAD

**Discussion by WaitingIdly** [ACGaming/UniversalTweaks#419] [Tweak Suggestions]
When you have very large smelteries, processing large amounts of items can lead to significant FPS lag when the smeltery is rendered, particularly when viewed from above. Adding a config to disable rendering items in the smeltery would be quite useful in saving FPS in these situations, or in packs where it is expected to automate the smeltery en mass.

see an honestly fairly mild example of how this can impact performance (yellow is looking up, and thus not rendered)
![image](https://github.com/ACGaming/UniversalTweaks/assets/25394029/65c395c1-567c-4ec9-af3a-e59d9bdbb780)

**Comment by ACGaming** [comment]
Implemented in 9c8de148f50c87a0a273b6b4786ec8028912f105
